index.js

```diff
@@ -0,0 +1,21 @@
+import { promises as fs } from 'fs';
+
+async function readFileAsync(filePath) {
+    try {
+        const data = await fs.readFile(filePath, 'utf8');
+        console.log(`${filePath}:`, data);
+        return data;
+    } catch (err) {
+        console.error('Error reading the file:', err);
+        throw err;
+    }
+}
+
+async function readFiles() {
+    const file1 = await readFileAsync('a.txt');
+    const file2 = await readFileAsync('b.txt');
+
+    console.log('file1 + file2 summary:', +file1 + +file2);

```

Comment:
I think there will be an 'NaN' if one of the files contains a string. It would be good to parse the data from files before performing calculations.